src/views/components/share-twitter-button.js

```diff
@@ -0,0 +1,21 @@
+import { h } from 'preact' /** @jsx h */
+
+export default ({ href, text, className }) => {
+  const handleClick = () => {
+    window.open(
+      `https://twitter.com/intent/tweet?text=${encodeURIComponent(text)}&url=${href}`,
```

Comment:
Provavelmente devo seguir o mesmo conceito do share pelo whatsapp usando o protocolo do app e.g. `twitter://`, essa pergunta do stackoverflow talvez responda..
http://stackoverflow.com/questions/11018416/open-twitter-app-in-html

Vou testar usando o protocolo `twitter://` pra ver o que pega, visto que, inicialmente o nosso app será acessado apenas via mobile. Devo criar uma estratégia pra identificar se for mobile usar o protocolo `twitter://` e se for desktop usar o link direto do twitter?